spec/lib/git_reflow/git_helpers_spec.rb

```diff
@@ -19,6 +19,23 @@ module Gitacular
     it      { expect{ subject }.to have_run_command_silently "git rev-parse --show-toplevel" }
   end
 
+  describe '.git_editor_command' do
+    subject { Gitacular.git_editor_command }
+    before { allow(ENV).to receive(:[]).with('EDITOR').and_return 'vim' }
```

Comment:
I would rather see this set explicitly to avoid issues this may pose with the CI: `ENV['EDITOR'] = 'vim'`